.env

```diff
@@ -4,7 +4,7 @@ REACT_APP_NETWORK_URL="https://dev-ethereum.knstats.com/v1/mainnet/geth?appId=de
 REACT_APP_MIGRATOR_ADDRESS=0x247B641bB4eAff621987E2B5c3D0247489556E75
 REACT_APP_MAINNET_ENV=staging
 REACT_APP_INFURA_KEY="9aa3d95b3bc440fa88ea12eaa4456161"
-REACT_APP_AGGREGATOR_API="https://dev-dmm-aggregator.knstats.com"
+REACT_APP_AGGREGATOR_API="https://aggregator-api.kyber.org"
```

Comment:
we should not change this. It's for dev environment

Reply:
for local development, If you want to run with production ENV, you can run the cmd yarn start-production @XiaoYhun :D